src/Module/Magic.php

```diff
@@ -116,11 +151,14 @@ protected function rawContent(array $request = [])
 				'acct:' . $user['nickname'] . '@' . $this->baseUrl->getHost() . ($this->baseUrl->getPath() ? '/' . $this->baseUrl->getPath() : '')
 			);
 
+			$this->logger->info('Headers: ', ['headers' => $header]);
+
 			// Try to get an authentication token from the other instance.
-			$curlResult = $this->httpClient->get($basepath . '/owa', HttpClientAccept::DEFAULT, [HttpClientOptions::HEADERS => $header]);
+			$curlResult = $this->httpClient->get($basepath . '/owa', HttpClientAccept::DEFAULT, [HttpClientOptions::HEADERS => $header, HttpClientOptions::ACCEPT_CONTENT => $accept_headers]);
```

Comment:
The "accept" value is the second parameter of the function. We should set the accept values there.